config/manifests/bases/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -4,63 +4,39 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
-    operators.operatorframework.io/builder: operator-sdk-v0.19.0
-    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+    categories: Logging & Tracing
+    certified: "false"
+    containerImage: quay.io/opentelemetry/opentelemetry-operator
+    createdAt: "2020-12-16T13:37:00+00:00"
+    description: Provides the OpenTelemetry components, including the Collector
+    operators.operatorframework.io/builder: operator-sdk-v1.2.0
```

Comment:
Is there a way to automatically update these when the sdk version changes? Or does this need to be added to some maintainers documentation somewhere?

Reply:
A new version of the SDK has just been released. I'll make sure to document it in a follow-up PR.